packages/block-library/src/video/edit.js

```diff
@@ -78,7 +78,7 @@ function VideoEdit( {
 	const posterImageButton = useRef();
 	const { id, caption, controls, poster, src, tracks } = attributes;
 	const prevCaption = usePrevious( caption );
-	const [ showCaption, setShowCaption ] = useState( !! caption );
+	const [ showCaption, setShowCaption ] = useState( caption.length > 0 );
```

Comment:
since most of these seem valid before this PR changes things up, should we go ahead and make a PR to update them now and get them out of this one?

Reply:
Yes agreed. There's even a few more changes now.

Reply:
Done in #56606